packages/calypso-typescript-config/js-package.json

```diff
@@ -7,9 +7,9 @@
 
 		"noEmit": true,
 
-		"lib": [ "ES2020", "DOM", "DOM.Iterable" ],
-		"module": "ES2020",
-		"target": "ES2020",
+		"lib": [ "ES2022", "DOM", "DOM.Iterable" ],
+		"module": "ES2022",
+		"target": "ES2022",
```

Comment:
Looks good to me 👍 We're already using ES2022 features, and our browser support policy is to support the latest versions, so I don't have concerns about this.